--- ./t/Mail-ClamAV.t.orig	2008-09-15 10:51:16.000000000 -0700
+++ ./t/Mail-ClamAV.t	2008-09-15 10:52:03.000000000 -0700
@@ -29,7 +29,6 @@
     CL_EACCES
     CL_ENULLARG
     CL_ETMPFILE
-    CL_EFSYNC
     CL_EMEM
     CL_EOPEN
     CL_EMALFDB
@@ -42,7 +41,6 @@
     CL_EIO
     CL_EFORMAT
     CL_ESUPPORT
-    CL_ELOCKDB
 
     CL_ENCINIT
     CL_ENCLOAD
--- ./ClamAV.pm.orig	2008-09-15 10:51:16.000000000 -0700
+++ ./ClamAV.pm	2008-09-15 10:52:52.000000000 -0700
@@ -58,7 +58,6 @@
     CL_EACCES
     CL_ENULLARG
     CL_ETMPFILE
-    CL_EFSYNC
     CL_EMEM
     CL_EOPEN
     CL_EMALFDB
@@ -71,7 +70,6 @@
     CL_EIO
     CL_EFORMAT
     CL_ESUPPORT
-    CL_ELOCKDB
 
     CL_ENCINIT
     CL_ENCLOAD
@@ -478,7 +476,6 @@
     if (strEQ("CL_EACCES", name)) return CL_EACCES;
     if (strEQ("CL_ENULLARG", name)) return CL_ENULLARG;
     if (strEQ("CL_ETMPFILE", name)) return CL_ETMPFILE;
-    if (strEQ("CL_EFSYNC", name)) return CL_EFSYNC;
     if (strEQ("CL_EMEM", name)) return CL_EMEM;
     if (strEQ("CL_EOPEN", name)) return CL_EOPEN;
     if (strEQ("CL_EMALFDB", name)) return CL_EMALFDB;
@@ -491,7 +488,6 @@
     if (strEQ("CL_EIO", name)) return CL_EIO;
     if (strEQ("CL_EFORMAT", name)) return CL_EFORMAT;
     if (strEQ("CL_ESUPPORT", name)) return CL_ESUPPORT;
-    if (strEQ("CL_ELOCKDB", name)) return CL_ELOCKDB;
 
     /* NodalCore */
     if (strEQ("CL_ENCINIT", name)) return CL_ENCINIT;
@@ -788,10 +784,6 @@
 
 tmpfile() failed 
 
-=item CL_EFSYNC
-
-fsync() failed 
-
 =item CL_EMEM
 
 memory allocation error 
@@ -840,10 +832,6 @@
 
 not supported data format 
 
-=item CL_ELOCKDB
-
-can't lock DB directory 
-
 =item CL_ENCINIT
 
 NodalCore initialization failed 
--- ./README.orig	2008-09-15 10:51:16.000000000 -0700
+++ ./README	2008-09-15 10:53:05.000000000 -0700
@@ -170,9 +170,6 @@
     CL_ETMPFILE
      tmpfile() failed
 
-    CL_EFSYNC
-     fsync() failed
-
     CL_EMEM
      memory allocation error
 
@@ -209,9 +206,6 @@
     CL_ESUPPORT
      not supported data format
 
-    CL_ELOCKDB
-     can't lock DB directory
-
     CL_ENCINIT
      NodalCore initialization failed
 
